webui/src/pages/namespace-detail/namespace-detail.tsx

```diff
@@ -0,0 +1,249 @@
+        this.setState({ truncateReadMore: false });
+    };
+
+    render(): React.ReactNode {
+        const { namespaceDetails, truncateReadMore } = this.state;
+        if (!namespaceDetails) {
+            return <>
+                <DelayedLoadIndicator loading={this.state.loading} />
+                {
+                    this.state.notFoundError ?
+                    <Box p={4}>
+                        <Typography variant='h5'>
+                            {this.state.notFoundError}
+                        </Typography>
+                    </Box>
+                    : null
+                }
+            </>;
+        }
+
+        const classes = this.props.classes;
+        return <>
+            <DelayedLoadIndicator loading={this.state.loading} />
+            <Box className={classes.head}>
+                <Container maxWidth='xl'>
+                    <Box className={classes.header}>
+                        <Grid container>
+                            <Grid item>
+                                <img src={namespaceDetails.logo || this.context.pageSettings.urls.extensionDefaultIcon}
+                                    className={`${classes.namespaceLogo} ${classes.badgePadding}`}
+                                    alt={namespaceDetails.displayName || namespaceDetails.name} />
+                            </Grid>
+                            <Grid item xs={7}>
+                                <Grid container spacing={2}>
+                                    <Grid item>
+                                        <Typography variant='h5'>{namespaceDetails.displayName || namespaceDetails.name}</Typography>
+                                    </Grid>
+                                </Grid>
+                                <Grid container spacing={2}>
+                                    <Grid item className={classes.descriptionPadding}>
+                                        {
+                                            namespaceDetails.description
+                                            ? truncateReadMore
+                                              ? <Truncate lines={2} ellipsis={<span>... <Link color='secondary' onClick={this.readMore}>Read more</Link></span>}>
+                                                { namespaceDetails.description }
+                                              </Truncate>
+                                              : <Typography>{ namespaceDetails.description }</Typography>
+                                            : null
+                                        }
+                                    </Grid>
+                                </Grid>
+                                <Grid container spacing={2}>
+                                    {
+                                        namespaceDetails.companyLink
+                                        ? <Grid item><Link color='secondary' href={namespaceDetails.companyLink}>{namespaceDetails.companyLink}</Link></Grid>
```

Comment:
Because this is always guaranteed to be a http/https link, can we do a little display magic to hide the protocol and make it more pleasant to read?

```suggestion
                                        ? <Grid item><Link color='secondary' href={namespaceDetails.companyLink}>{namespaceDetails.companyLink.replace(/https?:\/\//, '')}</Link></Grid>
```